src/main/java/com/krishagni/openspecimen/msk2/importer/CarsCpDetail.java

```diff
@@ -0,0 +1,117 @@
+package com.krishagni.openspecimen.msk2.importer;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.krishagni.catissueplus.core.biospecimen.events.CollectionProtocolDetail;
+import com.krishagni.catissueplus.core.biospecimen.events.CollectionProtocolEventDetail;
+import com.krishagni.catissueplus.core.biospecimen.events.CollectionProtocolSiteDetail;
+import com.krishagni.catissueplus.core.biospecimen.events.SpecimenRequirementDetail;
+
+public class CarsCpDetail {
+	private String cpShortTitle;
```

Comment:
I don't like introducing unwanted interaction objects... Please see my comment in importer for the main loop.